keyboards/hawtkeys/shortyk1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,88 @@
+// Copyright 2025 Hawtkeys
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        LT(0,KC_NO),
+        KC_A, KC_B, KC_C
+    ),
+    [1] = LAYOUT(
+        LT(0,KC_NO),
+        KC_1, KC_2, KC_3
+    ),
+    [2] = LAYOUT(
+        LT(0,KC_NO),
+        KC_Q, KC_W, KC_E
+    ),
+    [3] = LAYOUT(
+        LT(0,KC_NO),
+        KC_R, KC_T, KC_Y
+    )
+};
```

Comment:
Using `LT(0,KC_NO)` here doesnt really make sense as you are only overriding the tap function, and the hold functionality wouldn't produce any effect.

```suggestion

enum custom_keycodes {
    LAYER_ROTATE = SAFE_RANGE,
};

const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    [0] = LAYOUT(
        LAYER_ROTATE,
        KC_A, KC_B, KC_C
    ),
    [1] = LAYOUT(
        LAYER_ROTATE,
        KC_1, KC_2, KC_3
    ),
    [2] = LAYOUT(
        LAYER_ROTATE,
        KC_Q, KC_W, KC_E
    ),
    [3] = LAYOUT(
        LAYER_ROTATE,
        KC_R, KC_T, KC_Y
    )
};
```